server/jetstream_versioning_test.go

```diff
@@ -177,6 +208,51 @@ func TestJetStreamSetConsumerAssetVersionMetadata(t *testing.T) {
 	}
 }
 
+func TestJetStreamSetConsumerAssetVersionMetadataRemoveDynamicFields(t *testing.T) {
```

Comment:
Test names matter since they need to plug into the CI/CD matching when running the tests in parallel.

Reply:
The test names are currently prefixed with `TestJetStream` so they are run as part of the `js_tests`. This file also contains the following on top:
```go
//go:build !skip_js_tests
// +build !skip_js_tests
```

Is this sufficient or does something need to be changed?